packages/mui-base/src/PreviewCard/Root/PreviewCardRoot.tsx

```diff
@@ -0,0 +1,138 @@
+'use client';
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import type { PreviewCardRootProps } from './PreviewCardRoot.types';
+import { PreviewCardRootContext } from './PreviewCardContext';
+import { usePreviewCardRoot } from './usePreviewCardRoot';
+import { CLOSE_DELAY, OPEN_DELAY } from '../utils/constants';
+
+function PreviewCardRoot(props: PreviewCardRootProps) {
+  const { delayType = 'rest', delay, closeDelay, animated = true } = props;
+
+  const delayWithDefault = delay ?? OPEN_DELAY;
+  const closeDelayWithDefault = closeDelay ?? CLOSE_DELAY;
```

Comment:
```suggestion
  const { delayType = 'rest', delay = OPEN_DELAY, closeDelay = CLOSE_DELAY, animated = true } = props;
```

Why not use the destructuring for assigning the defaults?

Reply:
The prop-types/docs api gen doesn't allow variables as defaults when destructuring

Reply:
Damn, I am adding it on the code-infra topics for next week.